site/guides/porch-ui-installation.md

```diff
@@ -0,0 +1,52 @@
+# Accessing the Configuration as Data UI
+
+The easiest way to access the Configuration as Data UI is running by a docker
+container on your local machine where you'll be able to access the UI with your
+browser. Running the container locally simplifies the overall setup by allowing
+the UI to use your local kubeconfig and Google credentials to access the GKE
+cluster with Porch installed. This guide will show you how to do this.
+
+## Prerequisites
+
+To access the Configuration as Data UI with a docker container, you will need:
+
+*   [Porch](guides/porch-installation.md) installed on a GKE cluster
+*   [kubectl](https://kubernetes.io/docs/tasks/tools/) targeting the GKE cluster
+    with Porch installed
+*   [git](https://git-scm.com/)
+*   [docker](https://docs.docker.com/get-docker/)
+
+## Running locally in a container
+
+First, clone the
+[kpt-backstage-plugins](https://github.com/GoogleContainerTools/kpt-backstage-plugins)
+repository.
+
+```sh
+git clone https://github.com/GoogleContainerTools/kpt-backstage-plugins.git kpt-backstage-plugins
```

Comment:
nit: may be remove kpt-backstage-plugins dir in the end ?

Reply:
Removed.